integration_test/third_party_apps_data/applications/redis/expected_logs.yaml

```diff
@@ -0,0 +1,7 @@
+log_entries:
+- log_name: "redis"
+  field_matchers:
+    jsonPayload.level: "\\*"
```

Comment:
Can you test for `severity` as well?

Reply:
I tried to test using `severity` and something in the way the tooling is built couldn't handle it. I saw an error like the following:

```
2021/09/29 15:14:57 Attempt 1 of debian-10 test of redis finished with err=QueryLog() failed: rpc error: code = InvalidArgument desc = Field 'severity' is of incorrect type. Regular expression matches can only be performed against string fields., retryable=false
=== CONT  TestThirdPartyApps/ops-agent/debian-10/redis
    third_party_apps_test.go:385: Non-retryable error: QueryLog() failed: rpc error: code = InvalidArgument desc = Field 'severity' is of incorrect type. Regular expression matches can only be performed against string fields.
```

Reply:
Probably because `severity` is an enum: https://cloud.google.com/logging/docs/reference/v2/rest/v2/LogEntry#LogSeverity.

Proposed a potential fix for the internal testing infra and filed a ticket for tracking: b/202287040